### PR TITLE
spec: document historical exchange and market candles

### PR DESCRIPTION
**Pull request by Bogdanp**

### CODE REVIEW

**Comment by Bogdanp** [comment]
[Extend exchange candle generation to all-time](https://trello.com/c/GlexrcSO/320-extend-exchange-candle-generation-to-all-time)

File: spec.yaml [hunk 1]
```diff
@@ -1837,15 +1851,23 @@ paths:
...
+        | `1d`     | no limit |
+        | `4h`     | 120 days |
+        | `1h`     | 30 days  |
+        | `30m`    | 14 days  |
+        | `5m`     | 3 days   |
+        | `1m`     | 1 day    |
+
+        If the difference between your `end` and `start` parameters
+        for the `interval` you're requesting exceeds the limit, then
+        you'll receive an error.
```
Comment (ngauthier) [inline spec.yaml]: Are aggregated market pair candles changed? I didn't realize you did that too.

Comment (Bogdanp) [reply spec.yaml]: Yup:

* https://github.com/nomics-crypto/nommer/pull/411/files#diff-429152cf9bfb8b252921826411282bc9
* https://api.nomics.com/v1/markets/candles?base=BTC&interval=1h&key=REDACTED&quote=USD&start=2019-01-01T00:00:00Z&end=2019-01-31T00:00:00Z

**Comment by ngauthier** [comment]
Sweet! 🐑 it!

**Pull request merged**